properties/P000102.md

```diff
@@ -12,14 +12,18 @@ refs:
   name: On semi-metric spaces (Wilson)
 ---
 
+A space whose topology is generated by a *neighborhood basis* of
+symmetric balls $B(x,\varepsilon)=\{y\in X:d(x,y)<\varepsilon\}$ for some symmetric $d$.
+That is, a subset $U \subseteq X$ is open iff for each $x \in U$,
+there is some $V\subseteq U$ containing $x$ such that for each $y\in V$,
+there is $\varepsilon > 0$ with $B(y,\varepsilon)\subseteq V$.
```

Comment:
Actually, is this quite enough to guarantee that $B(x,\varepsilon)$ is a neighborhood of each $x$?

Reply:
No, this is equivalent to just saying every point having a neighborhood ball around it, which is symmetrizable, not semimetrizable.